docs/excel/excel-add-ins-comments.md

```diff
@@ -197,6 +197,105 @@ Excel.run(function (context) {
 });
 ```
 
+## Comment events
+
+Your add-in can also listen for comment additions, changes, and deletions. [Comment events](/javascript/api/excel/excel.commentcollection#event-details) occur on the `CommentCollection` object. To listen for comment events, first register the `onAdded`, `onChanged`, or `onDeleted` comment event handler. When a comment event is detected, use this event handler to retrieve data about the added, changed, or deleted comment. 
+
+### Comment addition events 
+The `onAdded` event is triggered when a new comment is added to the comment collection. The following sample shows how to register the `onAdded` event handler and then use this event handler to log the added comment's metadata. This sample assumes only a single comment has been added. 
```

Comment:
Nit: consider framing the "This sample assumes only a single comment has been added" line as a note. There are other places in the docs we say "Note that this sample..." and I think that similar verbage is warranted here. I also wonder if it's worth pointing out the subtle distinction that it's not so much that the sample "assumes" a behavior, but rather it only works for a single comment addition.